### DISCUSSION TITLE
How to diagnose sandboxed app failures?

### DISCUSSION THREAD

**Discussion by typpos** [sandboxie-plus/Sandboxie#1754] [Documents and notes]
Is there any guide on how an ordinary mortal (with dev skills) can diagnose why some application fail when sandboxed?

I know there is Sandboxie > File > Resource Access Monitor, but that hasn't helped me much. I've tried to review logs created with SysInternals tools like procmon but can't extract the relevant parts. Ability to narrow down issues without understanding the innards of Windows Kernel programming or running sandboxie in a debugger would be great.

Case in hand: I keep having hit and miss experiences with Unity-based games. Some work fine, some start and then freeze. Can't figure out why, or what would need to be changed in settings or sandboxie itself.

**Comment by DavidXanatos** [comment]
Yea I should make a guide, especially that the newer builds come with a lot of dangerous security breaking debug options that shouldn't be used by ordinary mortals :D
